src/glfw-sandbox/logger.cpp

```diff
@@ -0,0 +1,45 @@
+#include "logger.hpp"
+#include <iostream>
+#include <string>
+#include "spdlog/spdlog.h"
+
+// work in progres, adding some basic stuff at the moment
+// https://github.com/gabime/spdlog
+
+
+
+void logger::trace(std::string text) {
```

Comment:
In Nico's voice: "Are you sure the method changes internal state of the object?"

Reply:
In Nico's voice: "Are you sure the parameter gets changed inside the method?"

Reply:
In Nico's voice: "Are you sure it is necessary to perform copy?"